r/NEWS.md

```diff
@@ -19,6 +19,17 @@
 
 # arrow 14.0.2.9000
 
+##  Minor improvements and fixes
```

Comment:
Seems like we should note the build changes we just merged under minor improvements